iothub/device/samples/XamarinSamples/Xamarin/Xamarin/MainPage.xaml.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using System.Collections.ObjectModel;
+using Xamarin.Forms;
+using Microsoft.Azure.Devices.Client;
+
+namespace Xamarin
+{
+    public partial class MainPage : ContentPage
+    {
+        // String containing Hostname, Device Id & Device Key in one of the following formats:
+        //  "HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>"
+        //  "HostName=<iothub_host_name>;CredentialType=SharedAccessSignature;DeviceId=<device_id>;SharedAccessSignature=SharedAccessSignature sr=<iot_host>/devices/<device_id>&sig=<token>&se=<expiry_time>";
+        // Either set the IOTHUB_DEVICE_CONN_STRING environment variable or within launchSettings.json:
+        private static string DeviceConnectionString = Environment.GetEnvironmentVariable("IOTHUB_DEVICE_CONN_STRING_XAMARIN");
+
+        private static int MESSAGE_COUNT = 5;
+        private const int TEMPERATURE_THRESHOLD = 30;
+        private static String deviceId = "MyXamarinDevice";
+        private static float temperature;
+        private static float humidity;
+        private static Random rnd = new Random();
+        private static TransportType transportType;
+
+        public ObservableCollection<string> SentMessagesList = new ObservableCollection<string>();
+        public ObservableCollection<string> ReceivedMessagesList = new ObservableCollection<string>();
```

Comment:
Please follow the coding standards with the variable names:
[https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)